website/docs/blueprints/data-analytics/binpacking-custom-scheduler-eks.md

```diff
@@ -0,0 +1,56 @@
+---
```

Comment:
Could you move this file under `resources` section? https://github.com/awslabs/data-on-eks/tree/main/website/docs/resources

Reply:
done